lib/int.fz

```diff
@@ -177,24 +168,12 @@ ints is
   # helper feature to init int via an i64
   from_i64(val i64) int is
     s sign := if val < 0 then minus else plus
-    n u32 := if val < 0 then (-val).as_u32 else val.as_u32
-    (res,_) :=
-      (0..31)
-        .reduce (lists.empty bool, n) (r, _ ->
-          (l, rest) := r
-          ([(rest & 1) = 1] ++ l, rest>>1)
-        )
-    int s (uint res)
+    n u64 := if val < 0 then (-val).as_u64 else val.as_u64
```

Comment:
`-val` will overflow for `val = i64.min`

Reply:
@fridis good catch. :-) not sure what is the best way to fix this? add a pre-condition that disallows this one negative i64 or handle this single case explicitly?